doctrine/doctrine-bundle/1.12/config/packages/prod/doctrine.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('doctrine', [
+        'orm' => [
+            'auto_generate_proxy_classes' => false,
+            'metadata_cache_driver' => [
+                'type' => 'pool',
+                'pool' => 'doctrine.system_cache_pool',
+            ],
+            'query_cache_driver' => [
+                'type' => 'pool',
+                'pool' => 'doctrine.system_cache_pool',
+            ],
+            'result_cache_driver' => [
+                'type' => 'pool',
+                'pool'
+                
+                 => 'doctrine.result_cache_pool',
+            ],
+        ],
+    ]);
+
+    $containerConfigurator->extension('framework', [
+        'cache' => [
+            'pools' => [
+                'doctrine.result_cache_pool' => [
+                    'adapter' => 'cache.app',
+                ],
+                'doctrine.system_cache_pool' => [
+                    'adapter' =>
+                    
+                    
+                     'cache.system',
+                ],
```

Comment:
looks like still an issue with the code style fixure here

Reply:
@TomasVotruba any hint what else rule I would need to activate to fix this array styles?

Reply:
I'd try try `SetList::SPACES` set.

Reply:
Looks better a lot better thx, but one thing is still unexpected with the alias value be wrapped into a new line:

```php
return static function (ContainerConfigurator $containerConfigurator): void {
    $containerConfigurator->extension('doctrine', [
        'orm' => [
            // ...
            'mappings' => [
                'App' => [
                    // ...
                    'prefix' => 'App\Entity',
                    'alias'
                     => 'App',
                ],
            ],
        ],
    ]);
};
```